tests/jest/aggregate-translations.test.js

```diff
@@ -6,12 +6,12 @@ const MemoryFileSystem = require('memory-fs');
 const i18nSupportedLocales = require('../../scripts/aggregate-translations/i18nSupportedLocales');
 const aggregateTranslations = require('../../scripts/aggregate-translations/aggregate-translations');
 
-global.console = { warn: jest.fn() };
+global.console = { warn: jest.fn(), log: jest.fn() };
 const defaultSearchPatterns = baseDirectory => ([
-  `${baseDirectory}${path.sep}translations`,
-  `${baseDirectory}${path.sep}node_modules${path.sep}**${path.sep}translations`,
-  `${baseDirectory}${path.sep}packages${path.sep}terra-*${path.sep}translations`,
-  `${baseDirectory}${path.sep}packages${path.sep}**${path.sep}node_modules${path.sep}**${path.sep}translations`,
+  `${baseDirectory}/translations`,
+  `${baseDirectory}/node_modules/*/translations`,
+  `${baseDirectory}/packages/terra-*/translations`,
+  `${baseDirectory}/packages/*/node_modules/*/translations`,
```

Comment:
Glob expects only `/` for search patterns so we don't need to use device specific path separators here.